cli/tests/test_config_command.rs

```diff
@@ -310,9 +310,9 @@ fn test_config_list_origin() {
     test-layered-key = "test-layered-val" # repo $TEST_ENV/repo/.jj/repo/config.toml
     user.name = "Test User" # env
     user.email = "test.user@example.com" # env
-    debug.commit-timestamp = "2001-02-03T04:05:11+07:00" # env
+    debug.commit-timestamp = "2001-02-03T04:05:11+07:00[+07:00]" # env
```

Comment:
This looks a bit weird (though it might be technically correct to show `+07:00` twice.) Maybe we can set TOML-compatible timestamp string instead of using jiff's default formatting?